### ISSUE TITLE
Vuetifyjs progress does appear when assigned to :html

### ISSUE THREAD

**Issue by o1lab** [realdah/vue-blockui#3]
Im using vuetify and want to display below html 

In template
`        <BlockUI message="" :html="html">  </BlockUI>`

In data:
`html: "<v-progress-circular :width=\"3\" indeterminate color=\"red\"></v-progress-circular>",`

Circular progress bar from vuetify does not show up - how can I fix this ?

**Comment by realdah** [comment]
hi O1lab, that's because as of now, this vue-blockui does not support **[slot](https://vuejs.org/v2/guide/components-slots.html)**. 


since **v-progress-circular** is actually another VUE component, you probably have to show it with your own logic.

I will check if i can modify it to support slot.

**Comment by realdah** [comment]
@o1lab I published a new version 1.1.7 which support slot, please update your package.json with this version.


now, you can use like below:

```
<BlockUI>
     <v-progress-circular :width="3" indeterminate color="red"></v-progress-circular>
</BlockUI>
```

Let me know if you have problem.

**Comment by o1lab** [comment]
@realdah - It works. Thank you for publishing a new version.
